lib/yardstick/rule_description/classifier.rb

```diff
@@ -0,0 +1,105 @@
+module Yardstick
+  # Rule description composed of tokens which can be formatted
+  class RuleDescription
+    # Token classification
+    module Classifier
+      # List of classifiers
+      class List
+        extend Forwardable
+        include Concord.new(:classifiers), Enumerable
```

Comment:
Did you mean to make this `Enumerable` and `Forwardable`? I don't see an `#each` method, and I don't see any usage of the `def_*` methods in this class or it's descendants.